### DISCUSSION TITLE
BlTouch active, no compile

### DISCUSSION THREAD

**Discussion by Chester547** [mriscoc/Ender3V2S1#426] [General]
Hi! when i active bltouch, i have the same error "Marlin\src\lcd\e3v2\proui\dwin.cpp:2008:7: error: 'babystep' was not declared in this scope" i can't fix that, maybe can help me with the fix? babysteps is enable, bltouch and autolevel bilinear but nothing works

**Comment by LegenDarius01** [comment]
Still need an answer to that question?

**Comment by mriscoc** [comment]
https://github.com/mriscoc/Ender3V2S1/issues/490
